src/backend/commons/common/src/main/java/com/tencent/bk/job/common/util/ip/IpUtils.java

```diff
@@ -225,6 +225,9 @@ private static String getFirstMachineIpPreferV4() {
         Map<String, String> ipv6Map = getMachineIPv6Map();
         if (!ipv6Map.isEmpty()) {
             String ipv6 = ipv6Map.values().iterator().next();
+            if (ipv6.contains("%")) {
+                ipv6 = ipv6.substring(0, ipv6.indexOf("%"));
```

Comment:
这里可以补充下注释，看下为啥需要做这个处理，因为大部分看代码的人并不了解这个背景
```
Because link-local and site-local addresses are non-global, it is possible that different hosts may have the same destination address and may be reachable through different interfaces on the same originating system. In this case, the originating system is said to be connected to multiple zones of the same scope. In order to disambiguate which is the intended destination zone, it is possible to append a zone identifier (or scope_id) to an IPv6 address.
```

Reply:
ok

Reply:
多行注释用/* */